packages/next-plugin-treat/index.js

```diff
@@ -0,0 +1,19 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const TreatPlugin = require('treat/webpack-plugin');
+
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    webpack(config, { isServer }) {
```

Comment:
Where does `webpack` come from here?

Reply:
It's an ES2015 [method definition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) shorthand. The implementation was heavily inspired by the official [`@next/mdx`](https://github.com/zeit/next.js/tree/canary/packages/next-mdx) plugin.

Reply:
Right... I completely read that wrong. Dunno if it's me or maybe it's too clever 😅